agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -1483,8 +1487,6 @@ void ap_manager_thread::handle_hostapd_attached()
 
             usleep(ACS_READ_SLEEP_USC);
         }
-    } else {
-        ap_wlan_hal->read_supported_channels();
```

Comment:
commit message should mention such changes.
something like:

> 
> read_supported_channels() used to reads from GET_HW_FEATURES channel report and save supported channel list .
> The problem is that in case of ACS report is used instead.
> Since supported channels container is cleaned before reading the acs report read_supported_channels() can be read out side the **acs_enabled** condition.